src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/PartATemplateReplacementServiceTest.kt

```diff
@@ -46,16 +53,36 @@ internal class PartATemplateReplacementServiceTest : ServiceTestBase() {
         recallType = RecallTypePartA(value = RecallTypeValue.FIXED_TERM.displayValue, details = "My details"),
         responseToProbation = "They have not responded well",
         isThisAnEmergencyRecall = "Yes",
+        selectedAlternativesMap = mapOf(
+          "warning_letter_details" to "We sent a warning letter on 27th July 2022",
+          "drug_testing_details" to "drugs test passed",
+          "increased_frequency_details" to "increased frequency",
+          "extra_licence_conditions_details" to "licence conditions added",
+          "referral_to_other_teams_details" to "referral to other team",
+          "referral_to_approved_premises_details" to "referred to approved premises",
+          "referral_to_partnership_agencies_details" to "referred to partner agency",
+          "risk_escalation_details" to "risk escalation",
+          "alternative_to_recall_other_details" to "alternative action"
+        )
       )
 
       val result = partATemplateReplacementService.mappingsForTemplate(partA)
 
-      assertThat(result.size).isEqualTo(5)
+      assertThat(result.size).isEqualTo(14)
       assertThat(result["custody_status"]).isEqualTo("Police Custody")
       assertThat(result["recall_type"]).isEqualTo("Fixed")
       assertThat(result["recall_type_details"]).isEqualTo("My details")
       assertThat(result["response_to_probation"]).isEqualTo("They have not responded well")
       assertThat(result["is_this_an_emergency_recall"]).isEqualTo("Yes")
+      assertThat(result["warning_letter_details"]).isEqualTo("We sent a warning letter on 27th July 2022")
+      assertThat(result["drug_testing_details"]).isEqualTo("drugs test passed")
+      assertThat(result["increased_frequency_details"]).isEqualTo("increased frequency")
+      assertThat(result["extra_licence_conditions_details"]).isEqualTo("licence conditions added")
+      assertThat(result["referral_to_other_teams_details"]).isEqualTo("referral to other team")
+      assertThat(result["referral_to_approved_premises_details"]).isEqualTo("referred to approved premises")
+      assertThat(result["referral_to_partnership_agencies_details"]).isEqualTo("referred to partner agency")
+      assertThat(result["risk_escalation_details"]).isEqualTo("risk escalation")
+      assertThat(result["alternative_to_recall_other_details"]).isEqualTo("alternative action")
```

Comment:
Is it possible to add a test for the 'None' with details route to this test class?